controllers/chat/sendSignedMessage.js

```diff
@@ -0,0 +1,79 @@
+const Validator = require('fastest-validator');
+const {StreamChat} = require('stream-chat');
+const {responseError, responseSuccess} = require('../../utils/Responses');
+const {CHANNEL_TYPE} = require('../../helpers/constants');
+
+const v = new Validator();
+
+/**
+ *
+ * @param {import("express").Request} req
+ * @param {import("express").Response} res
+ */
+const sendSignedMesage = async (req, res) => {
+  const schema = {
+    channelId: 'string|empty:false',
+    message: 'string|empty:false',
+    channelType: 'number|empty:false'
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The validation schema is well defined and checks for the necessary fields in the request body. However, it would be better to also validate the `channelType` against the possible values defined in `CHANNEL_TYPE`. This will ensure that an invalid `channelType` is caught early during validation.

```diff
  const schema = {
    channelId: 'string|empty:false',
    message: 'string|empty:false',
-   channelType: 'number|empty:false'
+   channelType: { type: "enum", values: Object.values(CHANNEL_TYPE), empty: false }
  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->